### DISCUSSION TITLE
Limit number of attachs

### DISCUSSION THREAD

**Discussion by DarkCoderJ** [filamentphp/filament#4425] [Help] [answered]
Is it possible limiting number of attachs in the relation manager? I mean, i have Teams and Players. I want to limit the number of players that i can attach to a Team. Is it possible?

```
->headerActions([
                //Tables\Actions\CreateAction::make(),
                Tables\Actions\AttachAction::make()
                    ->preloadRecordSelect()
                    ->before(function () 
                    {
                     //  Same code here that don't allow to attach more than x players.
                    })
            ])
```

Thanks

**Comment by danharrin** [comment, marked as answer]
How about just a `->hidden(fn (RelationManager $livewire) => $livewire->getRelationship()->count() > 4)`

**Reply by DarkCoderJ** [reply]
That was what I was looking for. Thank you !!!
